src/test/integration/test_integration.py

```diff
@@ -54,7 +54,7 @@ def test_get_children(self):
         body = resp.json()
         result = body['result'][0]
         self.assertEqual(len(result['results']), 20)
-        self.assertEqual(result['total_count'], 21)
+        self.assertTrue(result['total_count'] > 20)
```

Comment:
Are these tests going against a live API or something, hence the reduced specificity?

Reply:
Tests go against CI for now